test/run-tests

```diff
@@ -650,7 +676,9 @@ def handle_task(gh, args, config, task, dry_run=False):
     target_url = None
     state = None
 
-    status_context = f"{config['name']}/{task.image['name']}"
+    status_context = get_status_context(
+        config["name"], task.image["name"], ansible_ver_str
```

Comment:
```suggestion
        config["name"], task.image["name"], ansible_version
```

Reply:
fixed in https://github.com/linux-system-roles/test-harness/pull/87/commits/a1f7e36ec876eb927bb6ddcd25ba619b13dc88c0